api/urls.py

```diff
@@ -25,12 +33,19 @@
 cart_router = routers.NestedDefaultRouter(router, "carts", lookup="cart")
 cart_router.register("items", views.CartItemViewSet, basename="cart-items")
 
+users_router = routers.NestedDefaultRouter(router, "users", lookup="users")
+# register_router = routers.NestedDefaultRouter(router, "registers", lookup="registers")
+
 # http://127.0.0.1:8000/api/products/a28a9e75-8c04-4754-a931-7f945ab33550/reviews/
 # http://127.0.0.1:8000/api/carts/
 
 urlpatterns = [
     path("", include(router.urls)),
     path("", include(perfume_router.urls)),
     path("", include(cart_router.urls)),
-    
```

Comment:
You've removed a line here, I dont which it is but it's creating conflicts, can you look at it and set it back?